internal/api/deployments.go

```diff
@@ -22,25 +22,25 @@ type Deployment struct {
 	WorkspaceID uuid.UUID `json:"workspace_id"`
 
 	ConcurrencyLimit       *int64                  `json:"concurrency_limit"`
-	ConcurrencyOptions     *ConcurrencyOptions     `json:"concurrency_options,omitempty"`
-	Description            string                  `json:"description,omitempty"`
+	ConcurrencyOptions     *ConcurrencyOptions     `json:"concurrency_options"`
```

Comment:
AFAIK, there's no need to specify `omitempty` on the Deployment object itself because it's not used in any API requests and is therefore never (un)marshalled by the `json` package.